string[] ids
furniture/Multiplanar_Model[] models
